### DISCUSSION TITLE
UndoManager, how to remove the trackSelection.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4226] [Q&A] [answered]
Hello @artf 
I think the undo manager has a strange behavior, he tracks the selection of a component at canvas. I know we don't have a good documentation of UndoManager, but, running the comand `editor.UndoManager.getConfig()` we have the `trackSelection` property.
![image](https://user-images.githubusercontent.com/2287371/160900384-f60d4909-3da7-4172-afec-af839239433e.png)

The question: Is it possible to change this property and not track the selection anymore?

**Comment by artf** [comment, marked as answer]
Yeah, you should be able to init the editor without tracking selection in undoManager.
```js
grapesjs.init({
 // ...
 undoManager: { trackSelection: false },
})
```
